web/package.json

```diff
@@ -10,16 +10,16 @@
     "dotenv-extended": "^2.0.0",
     "enzyme": "^2.8.2",
     "if-env": "^1.0.0",
+    "jsonwebtoken": "^8.1.0",
+    "jwt-decode": "^2.2.0",
```

Comment:
`jwt-decode` is in the wrong service. Remove from web and add to to graphql service.

Reply:
done